flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/CommonUtils.java

```diff
@@ -244,4 +246,17 @@ public static String getVariableName(Node node) {
         }
         return node.toString().strip();
     }
+
+    /**
+     * Get the expression node from the braced expression.
+     * @param bracedExpressionNode the braced expression node
+     * @return the expression node
+     */
+    public static ExpressionNode getExpressionNodeFromBracedExpression(BracedExpressionNode bracedExpressionNode) {
+        ExpressionNode exception = bracedExpressionNode.expression();
+        if (exception.kind() == SyntaxKind.BRACED_EXPRESSION) {
+            return getExpressionNodeFromBracedExpression((BracedExpressionNode) exception);
+        }
+        return exception;
+    }
```

Comment:
Isn't this redundant when using the visitor design pattern?

Reply:
updated